content/articles/external-dns-providers.markdown

```diff
@@ -0,0 +1,70 @@
+---
+title: External DNS Providers
+excerpt: Link an external DNS provider to your DNSimple account to manage zones at other authoritative DNS providers within DNSimple.
+categories:
+- DNS
+- Integrations
+---
+
+# External DNS Providers
+
+DNSimple supports the ability to view and manage zones that are deployed in other DNS providers external to DNSimple. DNSimple calls such providers "external DNS providers" and zones ["external zones"](/articles/managing-external-zones).
+
+## Supported Features
+
+- **Import external zones**: When you link an external DNS provider to your DNSimple account, all the zones hosted on that external DNS provider will be imported into DNSimple and listed on the [Zones](/articles/managing-external-zones) page.
+- **Management of external zone records**: List, create, update, and delete external zone records from DNSimple using the [Deployment Editor](/articles/deployment-editor). (Note: The list of supported record types differs based on the provider)
+- **2-way Syncing of Records**: Sync your external zone records from the external zone to DNSimple, or from DNSimple to the external zone, with the [Deployment Editor](/articles/deployment-editor#record-syncing).
+
+## Supported External DNS Providers
+
+To manage [external zones](/articles/managing-external-zones) from DNSimple, you first have to link an external DNS provider to your DNSimple account.
+
+DNSimple supports the management of zones from the authoritative DNS providers listed below.
+
+### Amazon Route 53
+
+DNSimple supports [Amazon Route 53](https://aws.amazon.com/route53/) as an external DNS provider.
+
+#### Prerequisites for linking Amazon Route 53 as an external DNS provider
+
+To link [Amazon Route 53](https://aws.amazon.com/route53/) as an external DNS provider, you need:
+
+- An [AWS access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for an AWS user with permission to manage public hosted zones at Route 53.
```

Comment:
Should we list the minimum permissions required? https://github.com/dnsimple/dnsimple-ops/issues/3193#issue-1710471000

Reply:
Added in https://github.com/dnsimple/dnsimple-support/pull/1032/commits/7c4857bd0a477c938380de52e06cdaec32b90d3d 🎉